src/squishy/core.clj

```diff
@@ -3,6 +3,8 @@
             [cemerick.bandalore :as sqs]
             [clojure.tools.logging :as log]))
 
+(def ^:dynamic *visibility-timeout* 30) ; seconds; the SQS default
```

Comment:
Should this be a dynamic var? Or a (potentially optional via a lower-arity) optional argument to `consume-message`. I'm naturally wary of dynamic vars. And I could see, if configuring this is something someone wants, possibly wanting different values per-queue.